distributed/deploy/ssh2.py

```diff
@@ -55,18 +68,21 @@ class Worker(Process):
     address: str
         The hostname where we should run this worker
     kwargs:
-        TODO
+        validate_host: bool
+            Validate host key is trusted (default is False)
     """
 
     def __init__(self, scheduler: str, address: str, **kwargs):
         self.address = address
         self.scheduler = scheduler
         self.kwargs = kwargs
+        self.validate_host = self.kwargs.get('validate_host', False)
```

Comment:
Let's pull this out from kwargs above

```python
    def __init__(self, scheduler: str, address: str, validate_host=False : bool, **kwargs):
```

Reply:
Or should we call this `known_hosts` and then just pass it through?

Or, adding even fewer opinions, have a `connect_kwargs=` keyword that we pass down to `asyncssh.connect`?